### ISSUE TITLE
Windows 7 and 10 install fails

### ISSUE THREAD

**Issue by thomasrivette** [PEM--/meteor-helper#55]
I have attempted to install meteor on Windows 7 and 10 with similar results.  I have suspect the path variable yet it appears to be more complex.

**Comment by PEM--** [comment]
Hi @thomasrivette, I don't own any Windows devices but some users have shared a solution for these environments: https://github.com/PEM--/meteor-helper#how-to-launch-meteor-on-windows-with-an-appropriate-path

**Comment by thomasrivette** [comment]
Thanks for your help!  Any other recommendations?


On Sun, Dec 20, 2015 at 12:05 PM, Pierre-Eric Marchandet <
notifications@github.com> wrote:

> Hi @thomasrivette <https://github.com/thomasrivette>, I don't own any
> Windows devices but some users have shared a solution for these
> environments:
> https://github.com/PEM--/meteor-helper#how-to-launch-meteor-on-windows-with-an-appropriate-path
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PEM--/meteor-helper/issues/55#issuecomment-166135475>.
>



-- 


Thomas Rivette

407 242 1263
thomas.rivette@gmail.com

**Comment by PEM--** [comment]
It didn't work for you? Try to launch Meteor on the command line (the Console, if my memory serves me well). If it fails, you should find some help on [Meteor's forum](https://forums.meteor.com).

**Comment by thomasrivette** [comment]
yes - not working.  I posted something on the forum.  the create ran
directly from the ".meteor" directory and created the simple-todo directory
and elements"css, html and js" in addition to the ".meteor" subdirectory.

when I cd to the simple-todo subdir and execute meteor it responds 'meteor'
is not recognized as an internal or external command.  which indicates that
the path i am able to view in th ht eadvanced setting and entered as
 c;\users\thomas rivette\appdata\local\" isn't working, which is what I
would expect as correct per the information posted.

Do you know where the log file is located?


On Sun, Dec 20, 2015 at 1:05 PM, Pierre-Eric Marchandet <
notifications@github.com> wrote:

> It didn't work for you? Try to launch Meteor on the command line (the
> Console, if my memory serves me well). If it fails, you should find some
> help on Meteor's forum <https://forums.meteor.com>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PEM--/meteor-helper/issues/55#issuecomment-166141663>.
>



-- 


Thomas Rivette

407 242 1263
thomas.rivette@gmail.com

**Comment by PEM--** [comment]
If the command doesn't start, only the OS can trigger a log. When `meteor` is launched, it simply displays its log on the STDOUT and STDERR of the process that has executed it.

Sounds like a `PATH` problem to me. Guys using the same OS as yours should be able to help you out better that I can. Let's hope that someone will help you out on Meteor's forum.

**Comment by thomasrivette** [comment]
I see forum dialog that introduces atom...not sure I understand the
implication as I am familiar with atom as an editor...

On Sun, Dec 20, 2015 at 1:34 PM, Pierre-Eric Marchandet <
notifications@github.com> wrote:

> If the command doesn't start, only the OS can trigger a log. When meteor
> is launched, it simply displays its log on the STDOUT and STDERR of the
> process that has executed it.
>
> Sounds like a PATH problem to me. Guys using the same OS as yours should
> be able to help you out better that I can. Let's hope that someone will
> help you out on Meteor's forum.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PEM--/meteor-helper/issues/55#issuecomment-166146060>.
>



-- 


Thomas Rivette

407 242 1263
thomas.rivette@gmail.com

**Comment by PEM--** [comment]
Actually, if you can't execute Meteor on the command line, there's nothing that Atom can do: it only means that the OS doesn't know where the `meteor` command is.

My guess is that most of the discussion about Atom on the forum are related to creating hybrid apps.

You should post a clear message: "I can't run Meteor on Windows 7 or 10" and explain what happens when you're are using `meteor create`, `cd` into it and launch `meteor`. Assuredly, someone has stumbled on this :wink:
